examples/voice-assistant/minimal_assistant.py

```diff
@@ -23,10 +23,11 @@ async def entrypoint(ctx: JobContext):
     assistant = VoiceAssistant(
         vad=silero.VAD.load(),
         stt=deepgram.STT(),
-        llm=openai.LLM(),
+        llm=openai.LLM.with_deepseek(),
```

Comment:
Let's keep the default to openai for the minimal_assistant :)

Reply:
sure, just add as example, will remove in next commit